src/i18n/i18n.ts

```diff
@@ -178,9 +178,18 @@ const dict = {
         canary: "Canary",
         settings: "Settings",
         decimal_separator: "Decimal Separator",
-        denomination_tooltip: "Choose your preferred denomination: BTC or sats",
-        decimal_tooltip:
-            "Choose your preferred decimal separator: dot or comma",
+        denomination_tooltip: "Choose your preferred denomination: btc or sat",
```

Comment:
Why lowercase BTC?

Reply:
I already fixed this here: https://github.com/BoltzExchange/boltz-web-app/pull/549/commits/941ab52e9c12b51b22e984beff91671ea4a5a3c4